src/domain/balances/entities/__tests__/balance.builder.ts

```diff
@@ -6,10 +6,15 @@ import type { Balance } from '@/domain/balances/entities/balance.entity';
 import { getAddress } from 'viem';
 
 export function balanceBuilder(): IBuilder<Balance> {
-  return new Builder<Balance>()
-    .with('tokenAddress', getAddress(faker.finance.ethereumAddress()))
-    .with('token', balanceTokenBuilder().build())
-    .with('balance', faker.string.numeric())
-    .with('fiatBalance', null)
-    .with('fiatConversion', null);
+  return (
+    new Builder<Balance>()
+      .with('tokenAddress', getAddress(faker.finance.ethereumAddress()))
+      .with('token', balanceTokenBuilder().build())
+      .with('balance', faker.string.numeric())
+      .with('fiatBalance', null)
+      .with('fiatConversion', null)
+      // TODO: Separate entity into base and extended to avoid doing this
+      // Transaction Service does not return this field and it will default to null
```

Comment:
Same as https://github.com/safe-global/safe-client-gateway/pull/2518/files#r2022886229

Reply:
Removed in d716924a75e14ee9d8cbb28e3a12f0923c181789.